teps/0052-nested-triggers.md

```diff
@@ -0,0 +1,97 @@
+set of interceptors to determine the event validitiy before invoking the desired resources.
+
+This also allows for shared data to be passed into multiple triggered `TaskRun` or `PipelineRun`
+objects. As a simple example, if shared volumes are desired for a set of tasks or pipelines, 
+today this would need to be done by passing the same data in the incoming event. 
+When processing is centralized, extensions can be set in the interceptor chain and 
+then used by all the downstream resources in processing.
+
+The result of centralizing this processing and passing common data down to downstream pipelines
+is a simpler experience for Triggers authors. It reduces the YAML burden of copying and maintaining
+the same interceptor chain across multiple triggers, it eases the barrier of entry for writing
+new triggers within the system, and it reduces noise in event listeners logs which is helpful
+for debugging issues.
+
+This solution is in line with multiple issues created by Tekton maintainers. We want to solve
+a broad set of use cases by allowing a complex graph of filters and invocations to be invoked
+by a single triggering event.
+
+* [tektoncd/triggers#820](https://github.com/tektoncd/triggers/issues/820)
+* [tektoncd/triggers#367](https://github.com/tektoncd/triggers/issues/367)
+* [tektoncd/triggers#44](https://github.com/tektoncd/triggers/issues/44)
+
+### Goals
+
+* Add a way for `EventListener` triggers and `Triggers` themselves to specify additional triggers
+  to process after interceptor logic concludes.
+* Provide a way to exclude some `Triggers` from processing by the `EventListener` HTTP handler
+* Pass data from interceptor chain processing down to the invoked `Triggers`. This is in line with
+  recent discussions around only allowing modifications to extensions, rather than the current format
+  where webhook extensions can modify the body and headers of the incoming event.
+
+### Non-Goals
+
+* While we posit that performance improvements are possible by limiting the number of invoked triggers,
+  significant performance gains is not intended to be a goal of this proposal.
+
+### Use Cases (optional)
+
+* Add common metadata to every github event based on external contextual information
+* Filter events that don't pass validation in a single Trigger to reduce noise and ease debugging
+* Generate a single PVC name for a set of pipelines to use based on event context and pass PVC 
+  into each pipelinerun as a workspace
+* Simplify the maintenance around branching logic in triggering. For example, if the logic for two
+  Triggers is similar, but dependent on a single parameter, those two pipelines can have the same
+  filter set defined in the upstream Trigger. Then, the two pipelines filters only need to specify
+  the differentiating logic.
+
+## Requirements
+
+* New `triggers` filed would be optional within `TriggerSpec` and accept any unicode character.
```

Comment:
I have the same comment as the Goals section. This should be more oriented towards the problem than the current solution e.g. 
1. Users should be able to reuse a single chain of interceptors across multiple triggers.

(I do get that this is a bit redundant with the Goals section so IMO its fine to skip this section at the moment and just the use Requirements section,)